x-pack/plugins/security_solution/public/explore/components/paginated_table/index.tsx

```diff
@@ -308,7 +308,7 @@ const PaginatedTableComponent: FC<SiemTables> = ({
                 sorting={tableSorting}
               />
               <FooterAction>
```

Comment:
We can add the justifyContent here: <FooterAction justifyContent="spaceBetween">